shift_management/security/ir.model.access.csv

```diff
@@ -0,0 +1,12 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink,perm_responsibility
+base_group_access_shift_coordinator,admin:access_shift_coordinator,model_nh_clinical_shift,nh_clinical.group_nhc_base,1,0,0,0,0
+admin_access_shift_coordinator,admin:access_shift_coordinator,model_nh_clinical_shift,nh_clinical.group_nhc_admin,1,1,1,1,0
+dev_access_shift_coordinator,dev:access_shift_coordinator,model_nh_clinical_shift,nh_clinical.group_nhc_dev,1,1,1,1,1
+wm_access_shift_coordinator,manager:access_shift_coordinator,model_nh_clinical_shift,nh_clinical.group_nhc_ward_manager,1,1,1,1,0
```

Comment:
This permissions is actually working fine to give the shift coordinator permissions for the shift but I agree that the naming is weird. That's because before this PR there was only permissions for the shift coordinator and I decided I wanted to add them for all user groups. The quickest way to do that was to copy them from an existing permissions file. I must have copied them from nh_eobs_slam_policy and then forgotten to change the actual names of the access rules.